packages/xsnap/src/text-shim.js

```diff
@@ -1,12 +1,21 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
I think we should this generally in `@agoric/eslint-config`. Not necessarily in this PR.